app/components/Utility/FeeAssetSelector.jsx

```diff
@@ -0,0 +1,304 @@
+import utils from "common/utils";
+import React from "react";
+import Immutable from "immutable";
+import counterpart from "counterpart";
+import AssetWrapper from "./AssetWrapper";
+import PropTypes from "prop-types";
+import {DecimalChecker} from "./DecimalChecker";
+import {Form, Input, Button, Tooltip} from "bitshares-ui-style-guide";
+import AssetSelect from "./AssetSelect";
+import {ChainStore} from "bitsharesjs";
+import SetDefaultFeeAssetModal from "../Modal/SetDefaultFeeAssetModal";
+import {debounce} from "lodash-es";
+import {connect} from "alt-react";
+import SettingsStore from "../../stores/SettingsStore";
+import {checkFeeStatusAsync} from "common/trxHelper";
+
+class FeeAssetSelector extends DecimalChecker {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            assets: [],
+            fee_amount: 0,
+            fee_asset_id:
+                ChainStore.assets_by_symbol.get(
+                    props.settings.get("fee_asset")
+                ) || "1.3.0",
+            fees: {},
+            feeStatus: {},
+            isModalVisible: false,
+            error: null
+        };
+        this._updateFee = debounce(this._updateFee.bind(this), 250);
```

Comment:
Should we rather debounce the whole component instead of those two calls?

Reply:
We should debounce actualy only updateFee, so I believe it will be ok to leave this one